ee/desktop/runtime/runner_test.go

```diff
@@ -92,7 +92,7 @@ func TestDesktopUserProcessRunner_Execute(t *testing.T) {
 
 			var logBytes threadSafeBuffer
 
-			r := New(log.NewLogfmtLogger(&logBytes), time.Second*1)
+			r := New(log.NewLogfmtLogger(&logBytes), time.Second*1, "some-where-over-the-rainbow.com")
```

Comment:
```suggestion
			r := New(log.NewLogfmtLogger(&logBytes), time.Second*1, "some-where-over-the-rainbow.example.com")
```

I feel humorless, but....